tests/integ/test_cte.py

```diff
@@ -64,7 +56,7 @@ def setup(request, session):
 
 
 def check_result(session, df, expect_cte_optimized):
-    df = df.sort(sql_expr("$1"))
+    df = df.sort(df.columns)
```

Comment:
what is this change for?

Reply:
this change was only sort first column earlier which was giving flaky results in sp environment. With this change, we are sorting all columns to ensure that all data is sorted correctly